tests/test_align.py

```diff
@@ -8,6 +8,30 @@
 
 @pytest.mark.bigdata
 class TestAlignMosaic(BaseHLATest):
+    """ Tests which validate whether mosaics can be aligned to an astrometric standard.
+
+        Characeteristics of these tests:
+          * A single astrometric catalog was obtained with both GAIA and non-GAIA
+            (PanSTARRS?) sources for the entire combined field-of-view using the GSSS
+            server.
```

Comment:
It would be good to confirm this information and where the extra information is coming from.

Reply:
Agreed.  I believe the best place to document this would be in the package's sphinx docs where we can point to external resources (from MAST, for example) to provide a complete description of what we are getting and the full provenance of that information.